src/openrct2-ui/windows/Ride.cpp

```diff
@@ -4271,28 +4302,19 @@ class RideWindow final : public Window
             break;
             case WIDX_ENTRANCE_STYLE_DROPDOWN:
             {
-                auto ddIndex = 0;
-                auto& objManager = GetContext()->GetObjectManager();
-                for (auto i = 0; i < MAX_STATION_OBJECTS; i++)
+                if (static_cast<std::vector<EntranceTypeLabel>::size_type>(dropdownIndex) >= _entranceStyleDropdownData.size())
```

Comment:
That is quite the line, just use size_t which all STL containers use

Reply:
Changed it.